src/data/pages/categorizing-new-discoveries.json

```diff
@@ -29,6 +29,7 @@
     {
       "title": "Planetary Orbital Sizes",
       "button": {
+        "icon": true,
```

Comment:
use `icon: true` to indicate that an icon displays, then the presence or absence of text to decide if it's an icon button or a text button with an icon.